exporters/metric/stdout/stdout.go

```diff
@@ -130,26 +133,39 @@ func InstallNewPipeline(config Config) (*push.Controller, error) {
 
 // NewExportPipeline sets up a complete export pipeline with the recommended setup,
 // chaining a NewRawExporter into the recommended selectors and batchers.
-func NewExportPipeline(config Config, period time.Duration) (*push.Controller, error) {
+func NewExportPipeline(config Config, period time.Duration, options ...push.Option) (*push.Controller, error) {
 	selector := simple.NewWithExactMeasure()
 	exporter, err := NewRawExporter(config)
 	if err != nil {
 		return nil, err
 	}
 	batcher := ungrouped.New(selector, exporter.config.LabelEncoder, true)
-	pusher := push.New(batcher, exporter, period)
+	pusher := push.New(batcher, exporter, period, options...)
 	pusher.Start()
 
 	return pusher, nil
 }
 
-func (e *Exporter) Export(_ context.Context, checkpointSet export.CheckpointSet) error {
+func (e *Exporter) encodeResource(resource *resource.Resource) string {
+	if e.encodedResource != nil {
+		return *e.encodedResource
+	}
+	e.encodedResource = new(string)
+	resourceSlice := export.LabelSlice(resource.Attributes())
```

Comment:
It seems like this pattern makes [this addition](https://github.com/open-telemetry/opentelemetry-go/blob/master/sdk/resource/iterator.go) unneeded. We should probably remove it.

Reply:
Yeah -- I was wishing for a more consolidated approach to Resource and metric.Labels. I'll mention it in a TODO.